ethereum/scripts/mint-nft.ts

```diff
@@ -0,0 +1,20 @@
+const hre = require("hardhat");
```

Comment:
Should use ES6 imports:

```suggestion
import { ethers, deployments, getUnnamedAccounts } from "hardhat";
```